src/hypothesis/extra/django/models.py

```diff
@@ -61,18 +57,25 @@ def field_mappings():
     global __default_field_mappings
 
     if __default_field_mappings is None:
+        # Auto and sized fields are handled in _get_strategy_for_field()
+        # URL and IPAddress fields are not yet handled
         __default_field_mappings = {
-            dm.SmallIntegerField: st.integers(-32768, 32767),
-            dm.IntegerField: st.integers(-2147483648, 2147483647),
             dm.BigIntegerField:
                 st.integers(-9223372036854775808, 9223372036854775807),
-            dm.PositiveIntegerField: st.integers(0, 2147483647),
-            dm.PositiveSmallIntegerField: st.integers(0, 32767),
             dm.BinaryField: st.binary(),
             dm.BooleanField: st.booleans(),
-            dm.DateTimeField: get_datetime_strat(),
+            dm.DateField: st.dates(),
+            dm.DateTimeField: st.datetimes(timezones=get_tz_strat()),
+            dm.DurationField: st.timedeltas(),
+            dm.EmailField: emails,
             dm.FloatField: st.floats(),
+            dm.IntegerField: st.integers(-2147483648, 2147483647),
             dm.NullBooleanField: st.one_of(st.none(), st.booleans()),
+            dm.PositiveIntegerField: st.integers(0, 2147483647),
+            dm.PositiveSmallIntegerField: st.integers(0, 32767),
+            dm.SmallIntegerField: st.integers(-32768, 32767),
+            dm.TimeField: st.times(timezones=get_tz_strat()),
+            dm.UUIDField: st.uuids(),
```

Comment:
The unnecessary line rearrangement makes it harder to review this section, because I have to work out what’s been added and what’s changed (or not).

The original ordering was fine, and kept all the integer-related strategies on adjacent lines.